contrib/quickstart/kratos/email-password/kratos.yml

```diff
@@ -2,6 +2,9 @@ version: v0.4.6-alpha.1
 
 dsn: memory
 
+rabbitmq:
+  url: amqp://exp-rabbitmq
```

Comment:
the same here, we should remove Expetitle-related names. Everything in shit repo should be ready for open-source

any ideas on how can we remove this line from the commit, without breaking the functionality?

Reply:
yes I will remove this completely